frame/state-trie-migration/src/lib.rs

```diff
@@ -1148,11 +1150,6 @@ mod mock {
 		sp_tracing::try_init_simple();
 		let mut ext: sp_io::TestExternalities = (custom_storage, version).into();
 
-		// set some genesis values for this pallet as well.
-		ext.execute_with(|| {
-			SignedMigrationMaxLimits::<Test>::put(MigrationLimits { size: 1024, item: 5 });
```

Comment:
this genesis did not happen in benchmarks, which allowed tests to pass, but live benchmarks to fail.

Removing this from genesis to ensure that such an issue wont be missed.